strongbox-web-core/src/test/java/org/carlspring/strongbox/config/IntegrationTest.java

```diff
@@ -3,40 +3,76 @@
 import org.carlspring.strongbox.MockedRemoteRepositoriesHeartbeatConfig;
 import org.carlspring.strongbox.app.StrongboxSpringBootApplication;
 import org.carlspring.strongbox.data.CacheManagerTestExecutionListener;
-import org.carlspring.strongbox.rest.common.RestAssuredTestExecutionListener;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
 import org.junit.jupiter.api.parallel.Execution;
-import org.junit.jupiter.api.parallel.ExecutionMode;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.ContextStoppedEvent;
+import org.springframework.context.event.EventListener;
 import org.springframework.security.test.context.support.WithUserDetails;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.context.WebApplicationContext;
+import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
 
 
 /**
  * Helper meta annotation for all rest-assured based tests. Specifies tests that
  * require web server and remote HTTP protocol.
  *
  * @author Alex Oreshkevich
+ * @author Przemyslaw Fusik
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @SpringBootTest(classes = { StrongboxSpringBootApplication.class,
                             MockedRemoteRepositoriesHeartbeatConfig.class,
+                            IntegrationTest.TestConfig.class,
                             RestAssuredConfig.class })
 @WebAppConfiguration("classpath:")
 @WithUserDetails(value = "admin")
 @ActiveProfiles(profiles = "test")
-@TestExecutionListeners(listeners = { RestAssuredTestExecutionListener.class,
-                                      CacheManagerTestExecutionListener.class }, mergeMode = TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS)
-@Execution(ExecutionMode.SAME_THREAD)
+@ContextConfiguration(locations = "classpath:/ldapServerApplicationContext.xml")
```

Comment:
@carlspring @fuss86 do we really need LDAP in all `s-w-core` tests?

it assumed to be used only in LDAP related tests like `LdapAuthenticatorConfigurationControllerTest`

Reply:
I don't think we do. I think that this is coming from a default context which applies to all of the tests.

Reply:
We have been trying to use single test context in s-w-c module. It still needs some work, we have bunch of tests failing, because of the unsatisfied context requirements.